functions/consumer/zeromq-consumer/src/main/java/org/springframework/cloud/fn/consumer/zeromq/ZeroMqConsumerProperties.java

```diff
@@ -30,13 +30,7 @@
     /**
      * The Topic to expose.
```

Comment:
This is wrong description for the property.
I think that one remove with the `SpEL` is much better.